assets.md

```diff
@@ -109,6 +110,14 @@ Link: https://grafana.com/orgs/otelsigsecurity
 
 The GitHub organization `open-telemetry-private` also exists for this purpose.
 
+### Netlify
+
+Link: https://app.netlify.com/login
+
+- Used to manage domain names under opentelemetry.io
+- Used to publish https://opentelemetry.io
+- Admins: [@austinlparker](https://github.com/austinlparker), [@svrnm](https://github.com/svrnm)
```

Comment:
@austinlparker anything else it's used for? and any other admins?

Reply:
@chalin is an Owner (along with a few other folks from CNCF)

Reply:
```suggestion
- Used to manage domain names under opentelemetry.io
- Used to publish https://opentelemetry.io
- Admins: [@austinlparker](https://github.com/austinlparker), [@svrnm](https://github.com/svrnm), [@chalin](https://github.com/chalin)
```